superset/assets/src/visualizations/deckgl/utils.js

```diff
@@ -89,8 +89,10 @@ export function getBuckets(fd, features, accessor) {
   breakPoints.slice(1).forEach((value, i) => {
     const range = breakPoints[i] + ' - ' + breakPoints[i + 1];
     const mid = 0.5 * (parseInt(breakPoints[i], 10) + parseInt(breakPoints[i + 1], 10));
+   //fix polygon cannot show
```

Comment:
Our linter is picky and wants a space after `//`, otherwise LGTM. You can run the linter locally with `npm run lint`